lab-collin/src/main.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { render as reactDomRender } from 'react-dom'; 
+import superagent from 'superagent';
+import './style/main.scss';
+
+
+class SearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchFormBoard: '',
+      searchFormLimit: 0,
+    };
+
+    this.handleBoardChange = this.handleBoardChange.bind(this);
+    this.handleLimitChange = this.handleLimitChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleBoardChange(event) {
+    this.setState({ searchFormBoard: event.target.value });
+  }
+
+  handleLimitChange(event) {
+    this.setState({ searchFormLimit: event.target.value });
```

Comment:
possible to set up conditional statement here to validate the `event.target.value` is between 0 and 100. Something you could try is declaring another `this.state` property with something like `limitError` and setting the default value to `false`. If the condition is not met, then set the value to `true`. Using a flag like this allows you to toggle attributes like the `className` on an element and you can use CSS to change the display.